gitlab2rbac.py

```diff
@@ -3,17 +3,23 @@
 import logging
 from collections import defaultdict
 from os import environ
-from time import sleep
+from time import sleep, time
 
 import kubernetes
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
 from gitlab import Gitlab
 from kubernetes.client.rest import ApiException
 from slugify import slugify
 
 logging.basicConfig(
-    format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO
+    format="%(asctime)s - %(levelname)s - %(message)s",
+    level=environ.get("LOGLEVEL", "INFO").upper(),
 )
 
+logger = logging.getLogger("gql")
+logger.setLevel(logging.WARNING)
```

Comment:
Maybe it would be safe to not assign the `gql` logger if you don't use it, someone may use it thinking it's gitlab2rbac's logger
```suggestion
logging.getLogger("gql").setLevel(logging.WARNING)
```